CODE_OF_CONDUCT.md

```diff
@@ -35,12 +35,12 @@ This Code of Conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting a project maintainer at info@open-book.org. All
+reported by contacting a project maintainer at joel@okuna.io. All
 complaints will be reviewed and investigated and will result in a response that
```

Comment:
@ThomasGaller we should not create a dependency on any one person, probably info@okuna.io is better, we need to create this email also for showing in the email from fields for system emails.

Reply:
Yes fully agree ;) I changed it to joel@.... because he used his email on the others cocs before so I though you wanted to use it ^^ For now i just updated the @open-book.org (this request is an exception ^^)

in general when Okuna is 'messaged' i think it should be info@... tho or press@... - a setup like that ^^